src/code/components/organism.js

```diff
@@ -1,6 +1,8 @@
 import React, {PropTypes} from 'react';
 
-const OrganismView = ({org, id, className="", width=200, flipped=false, style={}, onClick, wrapper }) => {
+const DEFAULT_DRAKE_HEIGHT = 200;
+
+const OrganismView = ({org, id, className="", height, width, flipped=false, style={}, onClick, wrapper }) => {
```

Comment:
The intention is to be able to create drakes based on height not just width. I've been hitting some issues using two dimensions instead of one, though.